src/main/scala/chisel3/stage/phases/Checks.scala

```diff
@@ -8,8 +8,8 @@ import firrtl.AnnotationSeq
 import firrtl.annotations.Annotation
 import firrtl.options.{Dependency, OptionsException, Phase}
 
-/** Sanity checks an [[firrtl.AnnotationSeq]] before running the main [[firrtl.options.Phase]]s of
-  * [[chisel3.stage.ChiselStage]].
+/** Sanity checks an [[firrtl.AnnotationSeq!]] before running the main [[firrtl.options.Phase]]s of
```

Comment:
```suggestion
/** Sanity checks an [[firrtl.AnnotationSeq]] before running the main [[firrtl.options.Phase]]s of
```
Weirdly the `!` doesn't seem to break it but it also doesn't seem right.